### ISSUE TITLE
[doc] perldelta list taking over the perl man page

### ISSUE THREAD

**Issue by jidanni** [Perl/perl5#21662]
$ `man perl `says
```
   History
           perlhist            Perl history records
           perldelta           Perl changes since previous version
           perl5341delta       Perl changes in version 5.34.1
           perl5340delta       Perl changes in version 5.34.0
           ...
```
Already 88 lines long and growing, making it the longest section of the
"perl" man page.

I would Just show a few lines and put the rest onto the perlhist page.

Or say
> The full list is too long, and already bloating the 'perl' man
> page. For a full list see ... or run the script on ... to generate it.

As a matter of fact they are also comprising lots of man pages
```
$ apropos perl|grep -c ^perl
213
$ apropos perl|grep ^perl5|grep -c delta
66
```
whereas other packages don't need to fill up the man page system with
changelog related stuff. So maybe move them somewhere else.

Other packages' man pages only contain current stuff, not history.

**Comment by Grinnz** [comment]
I agree, the perldelta list would be better moved to another page. The docs audit had suggested a dedicated page for historical changelogs. I would also consider moving them to perlhist.
